fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarView.kt

```diff
@@ -171,13 +173,18 @@ class BrowserToolbarView(
                     isPrivate = customTabSession.content.private,
                 )
             } else {
+                val shouldHideTabCounter = IncompleteRedesignToolbarFeature(context.settings()).isEnabled &&
```

Comment:
I am not sure if we need to check both flags here, I see them being used separately. Generally speaking, if we want to enable something for dev build only, the `Incomplete` flag should be enough for that, right? We don't really care about the `Complete` flag in that case. Also, on the implementation level, the `Incomplete` flag is on for dev builds, same as `Complete` – the only difference is that `Complete` is on nightly as well. But on nightly it can't be set to true, because the `Incomplete` flag can be set to true only on dev builds right now, so there is little sense to depend on the `Complete` flag here. Somehow, it feels to much of an explanation lol

Reply:
ahh I see what you mean, I'll make it just check the `Incomplete` flag and when the entire dev branch moves to `main`, I'll switch it to the `Complete` Flag.